src/sentry/workflow_engine/types.py

```diff
@@ -12,13 +12,22 @@
 T = TypeVar("T")
 
 
+class ActionType(StrEnum):
+    NOTIFICATION = "SendNotificationAction"
+    TRIGGER_WORKFLOW = "TriggerWorkflowAction"
+
+
 class DetectorPriorityLevel(IntEnum):
     OK = 0
     LOW = PriorityLevel.LOW
     MEDIUM = PriorityLevel.MEDIUM
     HIGH = PriorityLevel.HIGH
 
 
+class DataSourceType(StrEnum):
+    SNUBA_QUERY_SUBSCRIPTION = "SnubaQuerySubscription"
```

Comment:
FYI - i was thinking of the data source type a little differently: https://github.com/getsentry/sentry/pull/82116/files

This will allow us to define what goes into the workflow_engine, outside of the workflow engine - making the system a bit easier to use.

Reply:
Ok, do you know when you'll have time to merge that?

Reply:
hoping to merge it today (as long as reviews look good), only have 1 more meeting before coding time 🎉 

i _think_ you won't need to block on this though, since it's really just enforcing whatever string is defined in the metric alert code. (this PR and then my other draft PR for the model changes are my top priorities once meetings are done)